webapp/src/components/tutorialValidators.tsx

```diff
@@ -1,19 +1,17 @@
 import { MarkedContent } from "../marked";
 import { getBlocksEditor } from "../app";
-import TutorialOptions = pxt.tutorial.TutorialOptions;
-import TutorialStepInfo = pxt.tutorial.TutorialStepInfo;
 import CodeValidator = pxt.tutorial.CodeValidator;
 import CodeValidatorMetadata = pxt.tutorial.CodeValidatorMetadata;
 import CodeValidationResult = pxt.tutorial.CodeValidationResult;
 import CodeValidationExecuteOptions = pxt.tutorial.CodeValidationExecuteOptions;
 
-const defaultResult: CodeValidationResult = {
+const defaultResult: () => CodeValidationResult = () => ({
```

Comment:
nit; avoiding returning the same instance, as that could be mutated accidentally / lead to issues. Also thought to just Object.freeze it but we don't use it much / that can be hard to debug afterwards as it doesn't error or anything on attempted mutations